doc/luasnip.txt

```diff
@@ -2722,7 +2723,57 @@ there already exists a `luasnip.log.old`, it will be deleted.
+
+
+- `history`: If true, Snippets that were exited can still be jumped back into.
+    As Snippets are not removed when their text is deleted, they have to be
+    removed manually via `LuasnipUnlinkCurrent` if `delete_check_events` is not
+    enabled (set to eg. `'TextChanged'`).
+- `update_events`: Choose which events trigger an update of the active nodes’
+    dependents. Default is just `'InsertLeave'`, `'TextChanged,TextChangedI'`
+    would update on every change.
+- `region_check_events`: Events on which to leave the current snippet if the
+    cursor is outside its’ 'region'. Disabled by default, `'CursorMoved'`,
+    `'CursorHold'` or `'InsertEnter'` seem reasonable.
+- `delete_check_events`: When to check if the current snippet was deleted, and
+    if so, remove it from the history. Off by default, `'TextChanged'` (perhaps
+    `'InsertLeave'`, to react to changes done in Insert mode) should work just
+    fine (alternatively, this can also be mapped using
+    `<Plug>luasnip-delete-check`).
+- `store_selection_keys`: Mapping for populating `TM_SELECTED_TEXT` and related
+    variables (not set by default).
+- `enable_autosnippets`: Autosnippets are disabled by default to minimize
+    performance penalty if unused. Set to `true` to enable.
+- `ext_opts`: Additional options passed to extmarks. Can be used to add
+    passive/active highlight on a per-node-basis (more info in DOC.md)
+- `parser_nested_assembler`: Override the default behaviour of inserting a
+    `choiceNode` containing the nested snippet and an empty `insertNode` for
+    nested placeholders (`"${1: ${2: this is nested}}"`). For an example
+    (behaviour more similar to vscode), check
+    here <https://github.com/L3MON4D3/LuaSnip/wiki/Nice-Configs#imitate-vscodes-behaviour-for-nested-placeholders>
+- `ft_func`: Source of possible filetypes for snippets. Defaults to a function,
+    which returns `vim.split(vim.bo.filetype, ".", true)`, but check
+    filetype_functions <lua/luasnip/extras/filetype_functions.lua> or the
+    docs <https://github.com/L3MON4D3/LuaSnip/blob/master/DOC.md#filetype_functions>
+    for more options.
+- `load_ft_func`: Function to determine which filetypes belong to a given
+    buffer (used for `lazy_loading`). `fn(bufnr) -> filetypes (string[])`. Again,
+    there are some examples in
+    filetype_functions <lua/luasnip/extras/filetype_functions.lua>.
+- `snip_env`: The global environment will be extended with this table in some
+    places, eg. in files loaded by the
+    lua-loader <https://github.com/L3MON4D3/LuaSnip/blob/master/DOC.md#lua-snippets-loader>.
+    Setting `snip_env` to `{ some_global = "a value" }` will add the global
+    variable `some_global` while evaluating these files. If you mind the
+    (probably) large number of generated warnings, consider adding the keys set
+    here to the globals recognized by lua-language-server or add `---@diagnostic   disable: undefined-global` somewhere in the affected files.
```

Comment:
Uh, I'm not sure what happened with this line – the source looks okay at least…

Reply:
No idea.. :|
Maybe a linebreak before `` `---@diag...` `` would fix it?
I guess vim-help-files can't deal with `` ` ` `` across a linebreak, so they're kept whole.
Anyone's guess why the following text is still on the same line :D

Reply:
Right, that might actually be it. I'll add that real quick

Reply:
Nice, that's it :+1: